arangograph/single-sign-on.md

```diff
@@ -25,17 +44,98 @@ can sync the information about user resources in real-time:
 
-Organization administrators can generate an API key for a specific organization.
-The API key consists of a key and a secret. Using this key and secret as the
-Basic Authentication Header in SCIM provisioning, you can access the APIs and
+ArangoGraph organization administrators can generate an API key for a specific organization.
+The API token consists of a key and a secret. Using this key and secret as the
+Basic Authentication Header (Basic Auth) in SCIM provisioning, you can access the APIs and
 manage the user resources.
 
-To learn how to generate a new API key, see the [API Keys](my-account.html#api-keys) section.
+To learn how to generate a new API key in the ArangoGraph Dashboard, see the
+[API Keys](my-account.html#api-keys) section.
 
 {% hint 'info' %}
 When creating an API key, it is required to select an organization from the
 list.
-{% endhint %}
\ No newline at end of file
+{% endhint %}
+
+## Configure SAML 2.0 and SCIM using Okta
+
+You can enable SSO for your ArangoGraph organization using Okta as an Identity
+Provider (IdP). For more information about Okta, please refer to the
+[Okta Documentation](https://help.okta.com/en-us/Content/index.htm?cshid=csh-index){:target="_blank"}.
+
+### Create the SAML app integration in Okta
+
+1. Go to Okta and select **Applications** from the left sidebar menu.
+2. Click **Create App Integration**.
+3. In the **Create a new app integration** dialog, select **SAML 2.0**.
+
+   ![ArangoGraph Create Okta App Integration](images/arangograph-okta-create-integration.png)
+4. In the **General Settings**, specify a name for your integration.
+
+   ![ArangoGraph Okta Integration Name](images/arangograph-okta-integration-name.png)
+5. Click **Next**.  
+6. Configure the SAML settings:
+   - For **Single sign-on URL**, use `https://auth.arangodb.com/login/callback?connection=ORG_ID`
+   - For **Audience URI (SP Entity ID)**, use `urn:auth0:arangodb:ORG_ID`
+   
+   {% hint 'info' %}
+   Replace **ORG_ID** with your organization identifier from the 
+   ArangoGraph Dashboard. To find your organization ID, go to the **User Toolbar**
+   in the top right corner, which is accessible from every view of the Dashboard,
+   and click **My organizations**.
+   {% endhint %}
+
+   ![ArangoGraph Okta SAML General Settings](images/arangograph-okta-saml-general-settings.png)
+7. In the **Attribute Statements** section, define the following custom attributes:
+   - email: `user.email`
+   - first_name: `user.firstName`
+   - last_name: `user.lastName`
+   - email_verified: `true`
+   - picture: `user.profileUrl`
+
+   ![ArangoGraph Okta SAML Attributes](images/arangograph-okta-saml-attributes.png)
+8. Click **Next**.
+9. In the **Configure feedback** section, select **I'm an Okta customer adding an internal app**.
+10. Click **Finish**. The SAML app integration is now created.
+
+### SAML Setup
+
+1. After the integration is created, the **SAML Signing Certificates** section 
```

Comment:
This isn't really a procedure. Can you please rewrite it a paragraph style with adding steps for the user actions only? And it's very important to provide concrete info on how to copy/share settings and certificate.